manifests/03-rbac-role-ns-openshift-config-managed.yaml

```diff
@@ -0,0 +1,21 @@
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: console-operator
+  namespace: openshift-config-managed
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - services
+  - events
+  - configmaps
+  - secrets
+  - pods
+  verbs:
+  - get
+  - list
+  - watch
+  - delete
+  - update
+  - create
```

Comment:
We're asking for a lot of unnecessary permissions here. We only should need permission to read config maps. (I definitely don't want the operator to be able to read secrets in this namespace if it's not needed.)

Reply:
I get this error if I do not give read access for secrets:
```Failed to list *v1.Secret: secrets is forbidden: User "system:serviceaccount:openshift-console-operator:console-operator" cannot list resource "secrets" in API group "" in the namespace "openshift-config-managed"
```

Reply:
I have remove the full CRUD for that ns

Reply:
You may also be able to use:

```
resourceNames:
- openshift-config
```
to only read that resource specifically, though the `lister` may complain (should test this).